jwst/source_catalog/source_catalog.py

```diff
@@ -235,6 +235,13 @@ def set_segment_properties(self):
         self._xpeak = segm_cat.maxval_xindex
         self._ypeak = segm_cat.maxval_yindex
 
+        # newer versions of photutils use 'versions' instead of 'version'
+        # so copy over the new 'versions' into 'version' and remove 'versions'
+        # to stay consistent with the old behavior that this code expects
+        if 'versions' in segm_cat.meta:
+            segm_cat.meta['version'] = segm_cat.meta['versions']
+            del segm_cat.meta['versions']
+
```

Comment:
Is there a more elegant way to do this by somehow storing either `version` or `versions` (whichever appears in the meta data) in a local variable (or something like that) that then gets used in subsequent code?  Not sure I like the idea of forcing the code to continue using the old entry forever.

Reply:
If the changed output is acceptable I can look at updating this pr to use either `version` or `versions` and produce a file that for all released versions of photutils would produce a file with `version` and for all new versions produce a file with `versions. @hbushouse is it preferable to match the new output of photutils or match the previous output?